pkg/node/bootstrap.go

```diff
@@ -261,7 +274,7 @@ func bootstrapNode(
 
 // wait till some peers are connected. returns true if all is ok
 func waitPeers(kad *kademlia.Kad) bool {
-	for i := 0; i < 60; i++ {
+	for i := 0; i < 2*60; i++ { // max 2 minutes
```

Comment:
it might be useful to understand if this is indeed the underlying issue that is creating the instability. 2 minutes is a lot of time to establish connections to 25 peers. there might be some other underlying issue that is causing the bootstrapper instability.

Reply:
this is not needed 👍